From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Luke Wilde <lukew@serenityos.org>
Date: Mon, 27 Feb 2023 20:43:34 +0000
Subject: [PATCH] Replace PTHREAD_MUTEX_ERRORCHECK with PTHREAD_MUTEX_NORMAL

We currently don't support the PTHREAD_MUTEX_ERRORCHECK mutex type.
---
 src/mthreading/MThreading_POSIX.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mthreading/MThreading_POSIX.cpp b/src/mthreading/MThreading_POSIX.cpp
index ddebafec32385b98b7e2a0a0a98c3a6558cf069a..1acd8ba3780a9dca435f6596c32e7767720316d5 100644
--- a/src/mthreading/MThreading_POSIX.cpp
+++ b/src/mthreading/MThreading_POSIX.cpp
@@ -295,7 +295,7 @@ static void CreateMutex(Mutex* ret)
   throw MDFN_Error(ene.Errno(), _("%s failed: %s"), "pthread_mutexattr_init()", ene.StrError());
  }
 
- if((ptec = pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_ERRORCHECK/*PTHREAD_MUTEX_NORMAL*/)))
+ if((ptec = pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_NORMAL)))
  {
   ErrnoHolder ene(ptec);
 
